lib/srv/regular/sshserver.go

```diff
@@ -1168,6 +1185,21 @@ func (s *Server) HandleNewChan(ctx context.Context, ccx *sshutils.ConnectionCont
 	channelType := nch.ChannelType()
 	if s.proxyMode {
 		switch channelType {
+		// Channels of type "tracing-request" are sent to determine if ssh tracing envelopes
+		// are supported. Accepting the channel indicates to clients that they may wrap their
+		// ssh payload with tracing context.
+		case tracessh.TracingChannel:
+			ch, _, err := nch.Accept()
+			if err != nil {
```

Comment:
Can we log this error [like in the other channel types](https://github.com/gravitational/teleport/blob/217f7240ad966b17abb9e041d4be5b9b6673998c/lib/srv/regular/sshserver.go#L1214)?